examples/panda/panda_goal_reaching/controllers/robot_supervisor_manager/checkConvergence.py

```diff
@@ -0,0 +1,23 @@
+import matplotlib.pyplot as plt
+import numpy as np
+def moving_average(x, w):
+    return np.convolve(x, np.ones(w), 'valid') / w
+
+fp=open("Episode-score.txt", "r")
```

Comment:
Might something like `exports/Episode-score.txt`

Reply:
Ok, I will change dest for <code>exports/Episode-score.txt</code>.